Community_Guidelines.md

```diff
@@ -0,0 +1,380 @@
+
+**❌ Not Acceptable:**
+- Cold-calling members via DM
+- Posting promotional content without context
+- Astroturfing or fake testimonials
+- Disparaging competitors
+- Recruiting without Admin approval
+
+## Content Sharing
+
+### Links and Resources
+
+**Acceptable:**
+- Blog posts (your own or others')
+- Documentation and how-to guides
+- Open source projects
+- Conference talks and recordings
+- Relevant news articles
+
+**Guidelines:**
+- Provide context - don't just drop links
+- Disclose if it's your own content
+- Use appropriate channels (#blog-posts, #conferences, etc.)
+- Don't link to paywalled content without mentioning it
+
+### Self-Promotion
+
+**Allowed (in moderation):**
+- Sharing your blog posts in #blog-posts
+- Conference talk announcements in #conferences
+- Open source project launches in #github
+- Job postings in #jobs-board (discussion in #jobs-chat)
+
+**Not Allowed:**
+- Repeated self-promotion
+- Off-topic promotion
+- Promotional content disguised as help
+
+### Screenshots and Recordings
+
+**Privacy Considerations:**
+- Blur or redact sensitive information (names, emails, IPs, etc.)
+- Don't share screenshots of DMs without permission
+- Don't share internal company information
+- Assume workspace conversations are public
+
+**Recording Policy:**
+- Don't record calls without consent
+- Don't share recordings of Slack calls externally
+- Admin team may record calls for training purposes (with notice)
+
+## Accessibility and Inclusion
+
+### Making Content Accessible
+
+- Add alt text to images when possible
+- Use descriptive link text, not "click here"
+- Follow channel threading norms
+- Avoid wall-of-text messages
+- Use headings in longer posts
+
+### Language and Communication
+
+- Use clear, simple language
+- Define acronyms on first use
+- Be patient with non-native English speakers
+- Avoid sarcasm - it doesn't translate well
+- Use emoji thoughtfully (some are culturally specific)
+
+### Neurodiversity
+
+- Some members may communicate differently
+- Be patient with communication styles
+- Assume good intent
+- Offer clarification without judgment
+
+## Topics to Approach Carefully
+
+### Generally Acceptable (But Be Respectful)
+
+- Apple platform technical discussions (primary purpose)
+- IT and systems administration
+- Career advice and development
+- Conference and meetup announcements
+- Industry news relevant to Mac admins
+
+### Requires Caution
+
+**Politics:**
+- There is a private #politics channel available
+- Request access by DMing an Admin (see placeholder channel #politics)
+- General channels: keep politics relevant to our work (tech policy, privacy laws)
+- Admins may intervene if discussions become disruptive
+
+**Religion:**
+- Respect all beliefs and non-beliefs
+- Keep discussions civil
+- Don't proselytize
+- Admins may intervene if discussions become contentious
+
+**Controversial Topics:**
+- Use content warnings for potentially triggering topics
```

Comment:
- What format (spoiler tags, "[CW: topic]", etc.?)
- What topics need them?
- Is this _enforced_ or _encouraged_?